cedar-lean/Cedar/Partial/Expr.lean

```diff
@@ -15,8 +15,8 @@
 -/
 
 import Cedar.Data
+import Cedar.Thm.Data.List
```

Comment:
@emina doesn't want files in `Cedar/Partial` to import files in `Cedar/Thm`

Reply:
Ok, if I need to for theorems where should I move those theorems? (And also curious what's the organizational reason?)